src/include/wt_internal.h

```diff
@@ -311,6 +311,8 @@ struct __wt_page_index;
 typedef struct __wt_page_index WT_PAGE_INDEX;
 struct __wt_page_modify;
 typedef struct __wt_page_modify WT_PAGE_MODIFY;
+struct __wt_pinned_list;
+typedef struct __wt_pinned_list WT_PINNED_LIST;
```

Comment:
I think this should include "chunkcache" in the name - at the moment it's slightly too similar to (e.g.) a pinned cache content list.

Reply:
Good catch! Updated in [3d32412](https://github.com/wiredtiger/wiredtiger/pull/9466/commits/3d324120aafbac3396cae8bee5251b166c32447e)